.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1676,6 +1680,18 @@ dynamodb_Scenario_PartiQLSingle:
             - description: Run a scenario that creates a table and runs PartiQL queries.
               snippet_tags:
                 - python.example_code.dynamodb.Scenario_PartiQLSingle
+    C++:
+      versions:
+        - sdk_version: 1
+          github: cpp/example_code/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - cpp.example_code.dynamodb.Scenario_PartiQL_Single
+                - cpp.example_code.dynamodb.scenario.createTable

```

Comment:
Are the create/delete/wait snippets supposed to be part of this scenario, or just the one above it?

Reply:
I did not initially structure this as 2 scenarios.  The create and delete table will be part of the scenario and referenced with an addition tag.